### PR TITLE
Refs #30247 - VM deletion defaults to false

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ aren't familiar with all the methods Foreman provides. Documentation is availabl
 locally under `https://<fqdn>/templates_doc`. This documentation will be gradually
 enhanced in future releases to cover all available methods and macros.
 
+#### VM deletion behavior change
+
+Until now, the default installation sets the `destroy_vm_on_host_delete` setting to true, meaning associate VM is automatically deleted upon host deletion. This cause a lot of pain for users, who did not expect such behavior and deleted their production machines, when they simply wanted to drop the Foreman definition. The default behavior now changes and VMs remain running on the hypervisor or in the cloud. This can cause problems in case, user pay for the VM resources. If you prefer to delete the VM automatically upon host deletion, switch this setting back to true.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md:21]: I think this might fit better in upgrade warnings?

**Comment by tbrisker** [comment]
@upadhyeammit can you take this pr and reopen it against the 2.2 upgrade warnings? I think @ares won't have time to get to it before RC1 is out

**Comment by tbrisker** [comment]
closing for #1663. Thanks @ares !

**Comment by ares** [comment]
Thank you!

**Pull request closed without merging**